monkey/infection_monkey/monkey.py

```diff
@@ -175,12 +180,8 @@ def start(self):
                     if monkey_tunnel:
                         monkey_tunnel.set_tunnel_for_host(machine)
                     if self._default_server:
-                        if self._network.on_island(self._default_server):
-                            machine.set_default_server(get_interface_to_target(machine.ip_addr) +
-                                                       (':' + self._default_server_port if self._default_server_port else ''))
-                        else:
-                            machine.set_default_server(self._default_server)
-                        LOG.debug("Default server for machine: %r set to %s" % (machine, machine.default_server))
+                        machine.server_list = WormConfiguration.command_servers
```

Comment:
How about setting `machine.server_list` to a string of servers here? Rename your method `build_server_string()` to `get_command_server_string()` and move it to `WormConfiguration` class. This line should be `machine.server_list = WormConfiguration.get_command_server_string()`

Reply:
Corrected